pkg/settings/feature_string.go

```diff
@@ -0,0 +1,42 @@
+// Code generated by "stringer -type=Feature"; DO NOT EDIT.
```

Comment:
Why do we need `String`? No usages, internal name not equal to description.

Reply:
Fixed